dev/docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: '3.1'
+
+services:
```

Comment:
Any documentation need to be updated? Mostly thinking about local dev, is it as simple as `docker-compose up -d` to use MySQL, but the file system is used like it’s currently if there isn’t a container running?

Reply:
so I didn't change anything regarding the usual dev environment.

In case you would like to use the mysql database, yeah you will have to run the `docker-compose` command and then to use the config file `config-mariadb.yaml` when running the `perses` binary. 

Do you think it should be documented somewhere @sjcobb ?

Reply:
Gotcha, that makes sense! Could maybe update this [docs/configuration.md](https://github.com/perses/perses/blob/main/docs/configuration.md) file if it's referring to the same config